lib/IRGen/IRGenSIL.cpp

```diff
@@ -4815,16 +4819,15 @@ void IRGenSILFunction::visitEndApply(BeginApplyInst *i, EndApplyInst *ei) {
 
   if (!isAbort) {
     auto resultType = call->getType();
+    Explosion e;
```

Comment:
This was likely some mis-merge from the previous PRs. Unfortunately, it requires proper non-trivial coroutine result to see the difference. Now this codepath is executed via tests.